.travis.yml

```diff
@@ -8,6 +8,7 @@ env:
   - NODE_ENV=development
 
 before_install:
+  - npm i -g npm
```

Comment:
Take this back out when 1.0.0 is published and we don't need to build from GitHub.

Reply:
Done.